tests/test_stiefel.py

```diff
@@ -84,7 +84,7 @@ def test_belongs_shape(self):
 
     def test_random_uniform_and_belongs(self):
         point = self.space.random_uniform()
-        result = self.space.belongs(point, tolerance=1e-4)
```

Comment:
Could you also change the kwarg `tolerance` to atol for consistency?

Reply:
Done !